### PR TITLE
Remove swagger-ui link, fix page.version variable

### PR DESCRIPTION
**Pull request by chenopis**
Remove swagger-ui for now, as per Issue #2220. Specify `{{page.fullversion}}` variable for `v.1.6.0` and `{{page.version}}` as `v1.6`.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/3185)

### CODE REVIEW

**Comment by chenopis** [comment]
@ahmetb PTAL. Thx!

File: docs/concepts/overview/kubernetes-api.md [hunk 1]
```diff
@@ -24,13 +24,13 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api_changes.md).
 
-## OpenAPI and Swagger definitions
+## OpenAPI definition
 
-Complete API details are documented using [Swagger v1.2](http://swagger.io/) and [OpenAPI](https://www.openapis.org/). The Kubernetes apiserver (aka "master") exposes an API that can be used to retrieve the Swagger v1.2 Kubernetes API spec located at `/swaggerapi`. You can also enable a UI to browse the API documentation at `/swagger-ui` by passing the `--enable-swagger-ui=true` flag to apiserver.
```
Comment (ahmetb) [inline docs/concepts/overview/kubernetes-api.md]: Let's not remove where people can find swagger specs. You can remove the next paragraph, but this and the one that goes like `Starting with kubernetes 1.4...`  are still valuable.

File: redirects.rb [hunk 2]
```diff
@@ -51,7 +51,7 @@
 REPO_TMPL = "https://github.com/kubernetes/kubernetes/tree/%s/%s/:splat"
 
 fixed_redirects = """# 301 redirects (301 is the default status when no other one is provided for each line)
-/third_party/swagger-ui		/kubernetes/third_party/swagger-ui/
+#/third_party/swagger-ui		/kubernetes/third_party/swagger-ui/
```
Comment (ahmetb) [inline redirects.rb]: just delete

File: docs/concepts/overview/kubernetes-api.md (hunk 1 above)
Comment (chenopis) [inline docs/concepts/overview/kubernetes-api.md]: OK, no prob.

File: redirects.rb (hunk 2 above)
Comment (chenopis) [inline redirects.rb]: Sure thing. Was just commenting it out "just in case." =)

**Comment by chenopis** [comment]
@ahmetb OK, I've incorporated your feedback.

File: docs/concepts/overview/kubernetes-api.md [hunk 3]
```diff
@@ -24,11 +24,11 @@ In our experience, any system that is successful needs to grow and change as new
...
 
-## OpenAPI and Swagger definitions
+## OpenAPI definition
 
 Complete API details are documented using [Swagger v1.2](http://swagger.io/) and [OpenAPI](https://www.openapis.org/). The Kubernetes apiserver (aka "master") exposes an API that can be used to retrieve the Swagger v1.2 Kubernetes API spec located at `/swaggerapi`. You can also enable a UI to browse the API documentation at `/swagger-ui` by passing the `--enable-swagger-ui=true` flag to apiserver.
 
-We also host a version of the [latest v1.2 API documentation UI](http://kubernetes.io/kubernetes/third_party/swagger-ui/). This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.
+We host a version of the [latest {{page.version}} API documentation UI](https://kubernetes.io/docs/api-reference/{{page.version}}/). This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.
```
Comment (ahmetb) [inline docs/concepts/overview/kubernetes-api.md]: This is updated with the latest release, so if you are using a different version of Kubernetes you will want to use the spec from your apiserver.

this no longer applies. You can just delete this paragraph as the /docs/api-reference is already shown on the top of Reference section.

File: docs/concepts/overview/kubernetes-api.md [hunk 4]
```diff
@@ -24,11 +24,11 @@ In our experience, any system that is successful needs to grow and change as new
...
 
 Starting with kubernetes 1.4, OpenAPI spec is also available at `/swagger.json`. While we are transitioning from Swagger v1.2 to OpenAPI (aka Swagger v2.0), some of the tools such as kubectl and swagger-ui are still using v1.2 spec. OpenAPI spec is in Beta as of Kubernetes 1.5.
```
Comment (ahmetb) [inline docs/concepts/overview/kubernetes-api.md]: can you please link to wherever this  `/swagger.json` is? I can't seem to find it easily. I think it's at https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/swagger.json

File: docs/concepts/overview/kubernetes-api.md [hunk 5]
```diff
@@ -24,11 +24,11 @@ In our experience, any system that is successful needs to grow and change as new
 
 What constitutes a compatible change and how to change the API are detailed by the [API change document](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/docs/devel/api_changes.md).
 
-## OpenAPI and Swagger definitions
+## OpenAPI definition
 
 Complete API details are documented using [Swagger v1.2](http://swagger.io/) and [OpenAPI](https://www.openapis.org/). The Kubernetes apiserver (aka "master") exposes an API that can be used to retrieve the Swagger v1.2 Kubernetes API spec located at `/swaggerapi`. You can also enable a UI to browse the API documentation at `/swagger-ui` by passing the `--enable-swagger-ui=true` flag to apiserver.
```
Comment (ahmetb) [inline docs/concepts/overview/kubernetes-api.md]: I can't find this `/swaggerapi` thing. We should link to it.

Comment (chenopis) [inline docs/concepts/overview/kubernetes-api.md]: Yeah, I've searched all the docs and redirect files, and I can't find where that should be pointing to.

File: docs/concepts/overview/kubernetes-api.md (hunk 3 above)
Comment (chenopis) [inline docs/concepts/overview/kubernetes-api.md]: Ok

File: docs/concepts/overview/kubernetes-api.md (hunk 5 above)
Comment (ahmetb) [inline docs/concepts/overview/kubernetes-api.md]: @pwittrock any ideas where the `/swaggerapi` is?

**Comment by chenopis** [comment]
Ok, once more w/ feeling.

**Comment by chenopis** [comment]
@ahmetb PTAL.

**Pull request merged**